source/fundamentals/crud/read-operations/distinct.txt

```diff
@@ -0,0 +1,117 @@
+========================
+Retrieve Distinct Values
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve distinct values for a
+specified field across a single collection.
+
+Sample Data
+~~~~~~~~~~~
+
+To run the examples in this guide, load the sample data into the
+``ratings`` collection of the ``tea`` database with the following
+snippet:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/distinctValues.go
+   :language: go
+   :dedent:
+   :start-after: begin insert docs
+   :end-before: end insert docs
+
+.. include:: /includes/fundamentals/automatic-db-coll-creation.rst
+
+Each document contains a rating for a type of tea that corresponds to
+the ``type`` and ``rating`` fields.
+
+Distinct
+--------
+
+To retrieve distinct values for a specified field across a single
+collection, pass the following parameters to the ``Distinct()``
+function:
+
+- The field name you want distinct values for 
+- A ``non-nil`` query filter specifying which documents to match
+
+.. tip::
+
+   If you specify an empty query filter, the function matches all the
+   documents in a collection.
+
+Modify Behavior
+~~~~~~~~~~~~~~~
+
+You can modify the behavior of the ``Distinct()`` function by
+passing in a ``DistinctOptions``. If you don't specify a
+``DistinctOptions``, the driver uses the default values for each
+option.
+
+The ``DistinctOptions`` type allows you to configure options
+with the following functions:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Function
+     - Description
+
+   * - ``SetCollation()``
+     - | The type of language collation to use when sorting results.  
+       | Default: ``nil``
+
+   * - ``SetMaxTime()`` 
+     - | The maximum amount of time in milliseconds to allow the query to run.
```

Comment:
```suggestion
     - | The maximum amount of time to allow the query to run.
```

`MaxTime` is a `time.Duration`. Callers may specify it in other time units, e.g. `2 * time.Second`

Reply:
Ah yes, apologies on forgetting to update here!